lxd/project/project.go

```diff
@@ -2,12 +2,32 @@ package project
 
 import (
 	"fmt"
+	"strings"
 )
 
+// Default is the string used for a default project.
+const Default = "default"
+
+// separator is used to delimit the project name from the suffix.
+const separator = "_"
+
 // Prefix Add the "<project>_" prefix when the given project name is not "default".
-func Prefix(project string, s string) string {
+func Prefix(project string, suffix string) string {
 	if project != "default" {
```

Comment:
Shouldn't this be `if project != Default` ?

Reply:
@monstermunchkin yes it absolutely should, will fix! :)